lib/rake/compiler_config.rb

```diff
@@ -0,0 +1,23 @@
+require "yaml"
```

Comment:
We want to defer requiring `yaml`: https://github.com/rake-compiler/rake-compiler/commit/7e149899ff2f3ed9e73201405a1ffe106a72f978